core/trino-spi/src/main/java/io/trino/spi/session/PropertyMetadata.java

```diff
@@ -120,7 +154,15 @@ public T getDefaultValue()
      */
     public boolean isHidden()
     {
-        return hidden;
+        return flags.contains(HIDDEN);
+    }
+
+    /**
+     * Returns whether the property is not to be copied in CREATE TABLE LIKE INCLUDING PROPERTIES statements
+     */
+    public boolean isIgnoredInCreateTableLikeIncludingProperties()
```

Comment:
nit: `isIgnoredInCreateTableLikeIncludingProperties` is rather long and tied more to the SQL syntax.
`isInheritable` is a more succinct method name while still being meaningful.

Reply:
I would argue for `isInheritable` even though the current usages of this method check always the non-inheritance.
We don't see "isNotReadable" methods in other APIs also for a good reason.
The code is easier readable when dealing with "isXXX" instead of "isNotXXX".